README.md

```diff
@@ -37,14 +37,91 @@ To turn your web pages into Frames, you need to add basic metadata to your page.
+
+The `link` action redirects the user to an external URL. You can use this action to redirect to a URL without sending a POST request to the frame server.
+
+Clients do not make a request to the frame server for link actions. Instead, they redirect the user to the `target` URL.
+
+## `mint`
+
+The `mint` action allows the user to mint an NFT. Clients that support relaying or initiating onchain transactions may enhance the mint button by relaying a transaction or interacting with the user's walletl. Clients that do not fall back to linking to an external URL.
+
+The target property must be a valid [CAIP-10](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-10.md) address, plus an optional token ID appended with a `:`.
+
+## `tx`
+
+The `tx` action allows a frame to send a transaction request to the user's connected wallet. Unlike other action types, tx actions have multiple steps.
+
+First, the client makes a POST request to the `target` URL to fetch data about the transaction. The frame server receives a signed frame action payload in the POST body, including the address of the connected wallet in the `address` field. The frame server must respond with a `200 OK` and a JSON response describing the transaction which satisfies the following type:
+
+
+```ts
+type TransactionTargetResponse {
+  chainId: string;
+  method: "eth_sendTransaction";
+  params: EthSendTransactionParams;
+}
+```
+
+### Ethereum Params
+
+If the method is `eth_sendTransaction` and the chain is an Ethereum EVM chain, the param must be of type `EthSendTransactionParams`:
+
+- `abi`: JSON ABI which **MUST** include encoded function type and **SHOULD** include potential error types. Can be empty.
+- `to`: transaction recipient
+- `value`: value to send with the transaction in wei (optional)
+- `data`: transaction calldata (optional)
+
+```ts
+type EthSendTransactionParams {
+  abi: Abi | [];
+  to: `0x${string}`;
+  value?: string;
+  data?: `0x${string}`;
+}
+```
+
+Example:
+
+```json
+{
+  "chainId": "eip155:1",                                // The chain ID of the transaction
+  "method": "eth_sendTransaction",                      // The method to call on the wallet
+  "params": {
+    "abi": [...],                                       // JSON ABI of the function selector and any errors
+    "to": "0x0000000000000000000000000000000000000001", // The recipient of the transaction
+    "data": "0x00",                                     // Transaction calldata
+    "value": "123456789",                               // Value to send with the transaction
+  },
+};
+```
+
+The client then sends a transaction request to the user's connected wallet. The wallet should prompt the user to sign the transaction and broadcast it to the network. The client should then send a POST request to the `post_url` with a signed frame action payload including the transaction hash in the `transactionId` field to which the frame server should respond with a `200 OK` and another frame.
```

Comment:
> The client should then send a POST request to the `post_url`

Maybe worth specifying that this would either be the button's `post_url`, the Frame's `post_url` or the Frame URL depending on which are set.

Reply:
I see what you mean. We don't want to repeat that detail everywhere. 

Maybe we have a short section called "post_url evaluation" that describes the criteria, and we just link to that every time we need to mention this detail.

Reply:
I added a section called "Determining the `post_url`" in commit [76473af](https://github.com/open-frames/standard/pull/4/commits/76473aff4e51045addc5ac631671da165c5a2375)

Open to suggestions for the name - "post_url evaluation" sounds like there is some operation that needs to be done on a given post_url as opposed to a choice